templates/extensions/the-graph/README.md.args.mjs

```diff
@@ -0,0 +1,303 @@
+
+> Note: You will only need to do this once.
+
+\`\`\`
+yarn local-create
+\`\`\`
+
+> You should see some output stating your Subgraph has been created along with a log output on your graph-node inside docker.
+
+Next we will ship our subgraph! You will need to give your subgraph a version after executing this command. (e.g. 0.0.1).
+
+\`\`\`
+yarn local-ship
+\`\`\`
+
+> This command does the following all in one… 🚀🚀🚀
+
+-   Copies the contracts ABI from the hardhat/deployments folder
+-   Generates the networks.json file
+-   Generates AssemblyScript types from the subgraph schema and the contract ABIs.
+-   Compiles and checks the mapping functions.
+-   … and deploy a local subgraph!
+
+> If you get an error ts-node you can install it with the following command
+
+\`\`\`
+npm install -g ts-node
+\`\`\`
+
+You should get a build completed output along with the address of your Subgraph endpoint.
+
+\`\`\`
+Build completed: QmYdGWsVSUYTd1dJnqn84kJkDggc2GD9RZWK5xLVEMB9iP
+
+Deployed to http://localhost:8000/subgraphs/name/scaffold-eth/your-contract/graphql
+
+Subgraph endpoints:
+Queries (HTTP):     http://localhost:8000/subgraphs/name/scaffold-eth/your-contract
+\`\`\`
+
+&nbsp;
+
+#### ✅ Step 3: Test your Subgraph ✅
+
+Go ahead and head over to your subgraph endpoint and take a look!
+
+> Here is an example query…
+
+\`\`\`
+  {
+    greetings(first: 25, orderBy: createdAt, orderDirection: desc) {
+      id
+      greeting
+      premium
+      value
+      createdAt
+      sender {
+        address
+        greetingCount
+      }
+    }
+  }
+\`\`\`
+
+> If all is well and you’ve sent a transaction to your smart contract then you will see a similar data output!
+
+#### ✅ Side Quest: Run a Matchstick Test ✅
+
+Matchstick is a [unit testing framework](https://thegraph.com/docs/en/developing/unit-testing-framework/), developed by [LimeChain](https://limechain.tech/), that enables subgraph developers to test their mapping logic in a sandboxed environment and deploy their subgraphs with confidence!
+
+The project comes with a pre-written test located in \`packages/subgraph/tests/asserts.test.ts\`
+
+To test simply type....
+
+\`\`\`
+yarn subgraph:test
+\`\`\`
+
+> This will run \`graph test\` and automatically download the needed files for testing.
+
+You should receive the following output.
+
+\`\`\`
+Fetching latest version tag...
+Downloading release from https://github.com/LimeChain/matchstick/releases/download/0.6.0/binary-macos-11-m1
+binary-macos-11-m1 has been installed!
+
+Compiling...
+
+💬 Compiling asserts...
+
+Igniting tests 🔥
+
+asserts
+--------------------------------------------------
+  Asserts:
+    √ Greeting and Sender entities - 0.102ms
+
+All 1 tests passed! 😎
+
+[Thu, 07 Mar 2024 15:10:26 -0800] Program executed in: 1.838s.
+\`\`\`
+
+> NOTE: If you get an error, you may trying passing \`-d\` flag \`yarn subgraph:test -d\`. This will run matchstick in docker container.
+
+&nbsp;
+
+## Shipping to TheGraph studio
```

Comment:
It is called "Subgraph Studio" not TheGraph Studio.

Reply:
Ohh my bad, updated at [40e28d3](https://github.com/scaffold-eth/create-eth/pull/25/commits/40e28d31bd8fca7c0b028bd7dbcbc375a425180e)